packages/server/src/threads/automation.ts

```diff
@@ -263,6 +263,13 @@ class Orchestrator {
         this.context.env = await sdkUtils.getEnvironmentVariables()
         this.context.user = this.currentUser
 
+        const { config } = await configs.getSettingsConfigDoc()
```

Comment:
Will this always exist? If so, should we make the fields not optional on the type?

Reply:
I'm going by the types entirely, I'm not sure.

Reply:
If the document exists the config should exist on it - I believe the type is correct as well, it is not optional on it. The get will fail and 404 the endpoint if the document doesn't exist, therefore the `{ config }` will always be safe.

Reply:
I _think_ I've implemented what you're suggesting, PTAL 🙏